src/interface/widgets/widget.cpp

```diff
@@ -258,15 +260,15 @@ void Visualizer::drawBall(QPainter &painter) {
 }
 
 // draw the robots
-void Visualizer::drawRobots(QPainter &painter) {
+void Visualizer::drawRobots(QPainter &painter, rtt::world_new::view::WorldDataView world) {
     // draw us
-    for (auto &robot : rtt::ai::world::world->getUs()) {
-        drawRobot(painter, *robot, true);
+    for (rtt::world_new::view::RobotView robot : world->getUs()) {
```

Comment:
const &?

Reply:
oh wait nvm these things are pointers right? Not sure.

Reply:
They are RobotViews, which only hold a single pointer to actual robot data. They wouldn't have to be const but I'll add it anyway for consistency